src/middlewares/AuthCheck.ts

```diff
@@ -0,0 +1,21 @@
+import UserService from "../services/UserService";
+import { IRequestHandler } from "../types/IRequestHandler";
+
+
+
+const AuthCheck = ( // eslint-disable-line @typescript-eslint/no-explicit-any
+): IRequestHandler  => 
+  async (req:any, res, next): Promise<any> => {
```

Comment:
We should not commit `@typescript-eslint/no-explicit-any` unless there's a very specific reason to do it.

We must avoid dealing with a `req: any` in our code.